src/pages/AccessManagement/PrivilegeTR.tsx

```diff
@@ -284,79 +281,29 @@ const PrivilegeTR: FC<PrivilegeTRProps> = (props) => {
 					</Box>
 				</td>
 			</tr>
-			<Modal
-				withinPortal
-				size="md"
-				opened={isDeletedRoleOpen}
-				onClose={handleCloseDelete}
-				title={'Delete Role'}
-				className={classes.modalStyle}
-				centered>
-				<TextInput
-					label="Are you sure you want to delete this Role?"
-					type="text"
-					onChange={(e) => {
-						setUserInput(e.target.value);
-					}}
-					placeholder={`Please enter the Role to confirm, i.e. ${roleName}`}
-					required
-					mb={20}
-				/>
 
-				<Group justify="right" mt={10}>
-					<Button
-						variant="filled"
-						color="gray"
-						className={classes.modalActionBtn}
-						disabled={UserInput === roleName ? false : true}
-						onClick={handleDelete}>
-						Delete
-					</Button>
-					<Button onClick={handleCloseDelete} variant="outline" color="gray" className={classes.modalCancelBtn}>
-						Cancel
-					</Button>
-				</Group>
-			</Modal>
+			<DeleteOrResetModal
+				type="delete"
+				isOpen={isDeletedRoleOpen}
+				onClose={handleCloseDelete}
+				header={'Delete Role'}
```

Comment:
Can directly pass as string.

Reply:
Updated these to be passed as string directly.